services/QuillLMS/client/app/bundles/Teacher/components/student_profile/student_profile_units.jsx

```diff
@@ -98,7 +98,7 @@ export default class StudentProfileUnits extends React.Component {
     if (loading) { return <LoadingIndicator /> }
 
     const content = this.displayedUnits().map(unit => {
-      const { unit_id, unit_name, } = unit[Object.keys(unit)[0]][0]
+      const { unit_id, unit_name, staggered_release_unit_status, } = unit[Object.keys(unit)[0]][0]
```

Comment:
all instances of `staggered_release_unit_status` need to be updated to `pack_sequence_item_status`